packs/backgrounds/mechanical-symbiosis.json

```diff
@@ -30,7 +30,121 @@
+                    {
+                        "label": "PF2E.SkillDip",
+                        "value": "diplomacy"
+                    },
+                    {
+                        "label": "PF2E.SkillItm",
+                        "value": "intimidation"
+                    },
+                    {
+                        "label": "PF2E.SkillMed",
+                        "value": "medicine"
+                    },
+                    {
+                        "label": "PF2E.SkillNat",
+                        "value": "nature"
+                    },
+                    {
+                        "label": "PF2E.SkillOcc",
+                        "value": "occultism"
+                    },
+                    {
+                        "label": "PF2E.SkillPrf",
+                        "value": "performance"
+                    },
+                    {
+                        "label": "PF2E.SkillRel",
+                        "value": "religion"
+                    },
+                    {
+                        "label": "PF2E.SkillSoc",
+                        "value": "society"
+                    },
+                    {
+                        "label": "PF2E.SkillSte",
+                        "value": "stealth"
+                    },
+                    {
+                        "label": "PF2E.SkillSur",
+                        "value": "survival"
+                    },
+                    {
+                        "label": "PF2E.SkillThi",
+                        "value": "thievery"
+                    }
+                ],
+                "flag": "entitySkill",
+                "key": "ChoiceSet",
+                "prompt": "GM-Selected Skill"
+            },
+            {
+                "key": "ActiveEffectLike",
+                "mode": "upgrade",
+                "path": "system.skills.{item|flags.pf2e.rulesSelections.entitySkill}.rank",
+                "value": 1
+            },
+            {
+                "domain": "skill-check",
+                "key": "RollOption",
+                "option": "entity-is-aiding",
+                "toggleable": true
+            },
+            {
+                "key": "FlatModifier",
+                "label": "Mechanical Symbiosis",
+                "predicate": [
+                    "entity-is-aiding"
+                ],
+                "selector": "{item|flags.pf2e.rulesSelections.entitySkill}",
+                "type": "circumstance",
+                "value": 1
+            },
+            {
+                "key": "Note",
+                "outcome": [
+                    "failure"
+                ],
+                "predicate": [
+                    "entity-is-aiding"
+                ],
+                "selector": "{item|flags.pf2e.rulesSelections.entitySkill}",
+                "text": "You become @UUID[Compendium.pf2e.conditionitems.Item.Stunned]{Stunned 1}",
+                "title": "Mechanical Symbiosis (Failure):"
```

Comment:
```suggestion
                "text": "PF2E.SpecificRule.MechanicalSymbiosis.FailureNote"
```
We need to localize this. To do this open up `/static/lang/re-en.json` and go under SpecificRule. Make a new object for `MechanicalSymbiosis` (be sure to keep the file alphabetized) and put entries in for the text used in the rules here. We don't need specific titles for these notes though, that can just use the item name as normal.

Reply:
One thing I'd like to keep labels for is since the majority of the time you're not rolling skill checks against a given DC, so both the failure and the critical failure notes will show up regardless of the actual degree of success, which means you get handed "You become Stunned 1" and "You become Stunned 2" without any context - is there any other solution for this?

Reply:
It will have a label from the item name automatically, pointing back to the source.

Reply:
Yes, what I'm saying is that 99% of the time the results will look like this: 
![image](https://github.com/foundryvtt/pf2e/assets/110754119/70f40d55-6383-4f5f-bff6-9ad644a8c5b4)

since normally skill checks will be rolled without a targeted DC, so the check usually won't know whether it's a success/failure, so it prints all notes regardless of success state.